deploy/operator.yaml

```diff
@@ -46,3 +46,5 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "marketplace-operator"
+            - name: RELEASE_VERSION
+              value: "0.0.1-snapshot"
```

Comment:
Is this the version we want to hardcode? What is our plan for incrementing `RELEASE_VERSION` in the future? We probably want to scope that out and define it in a doc somewhere.

Reply:
The string "0.0.1-snapshot" is substituted by the CVO with the version of the payload
OpenShift operators are expected to report the payload version as their own and developers will have map from payload to code version.

Reply:
+1